lib/ui/settings/data/export_widget.dart

```diff
@@ -145,9 +148,15 @@ Future<void> _exportCodes(BuildContext context, String fileContent) async {
 
 Future<String> _getAuthDataForExport() async {
   final codes = await CodeStore.instance.getAllCodes();
-  String data = "";
+  List items = [];
   for (final code in codes) {
-    data += code.rawData + "\n";
+    items.add({
+      "code": code.rawData,
+      "pinned": code.isPinned,
```

Comment:
note: Because we had modified rawData, here we were exporting same information twice.
Also, viewQrCode was broken because we had changes the contract for the rawData format.